### DISCUSSION TITLE
reply/react to notifications

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#46] [Enhancements]
Some apps give users the option to reply in the notification without switching context to the app that notified.

having a reply or common reactions to a notification, would be nice.

**Comment by chetanyakan** [comment]
Mattermost mobile app does have this feature.
But the content of the notification can be hidden in the system console settings, in which case, you see a notification: "A sent you a message" and you see an option to reply to that notification, which doesn't make much sense.

@enahum Can we remove the reply option if the content of the notification is hidden?

**Reply by Willyfrog** [reply]
It would totally make sense to remove it

**Reply by enahum** [reply]
I believe with a bit of work this can be done. It will require changes at the server / push proxy & the mobile app to be effective and accurate

**Reply by esethna** [reply]
+1, that's definitely a UX flaw. @chetanyakan feel free to contribute that fix ;)
